frontend/app/templates/work_packages/activities/_user.html

```diff
@@ -27,23 +27,34 @@
   <span class="date">{{ isInitial ? I18n.t('js.label_created_on') : I18n.t('js.label_updated_on') }} <op-date-time date-time-value="activity.props.createdAt" /></span>
   <span class="user-comment wiki">
     <div ng-if="inEdit">
-      <form name="editCommentForm">
-        <div>
-          <textarea id="edit-comment-text"
-                    name="edit_comment_text"
-                    class="add-comment-text"
-                    ng-model="activity.props.comment.raw"
-                    ng-bind-html="activity.props.comment.html"
-                    rows="4"
-                    required>
-          </textarea>
+      <form class="user-comment--form inplace-edit--write-value">
+        <div class="textarea-wrapper" ng-class="{'-preview': inPreview}">
+            <textarea wiki-toolbar
+                      class="edit-comment-text focus-input inplace-edit--textarea -autogrow"
+                      ng-hide="inPreview"
+                      preview-toggle="toggleCommentPreview()"
+                      ng-model="activity.props.comment.raw"
+                      ng-bind-html="activity.props.comment.html"
```

Comment:
You are directly binding to the structure of the server's JSON response here. A model layer would remove the dependency.